projects/plugins/jetpack/class.jetpack.php

```diff
@@ -4362,15 +4362,20 @@ function ( $domains ) {
 						exit;
 					}
 
-					$redirect_url = self::admin_url(
-						array(
-							'page'     => 'jetpack',
-							'action'   => 'authorize_redirect',
-							'dest_url' => rawurlencode( $dest_url ),
-							'done'     => '1',
-						)
+					$redirect_args = array(
+						'page'     => 'jetpack',
+						'action'   => 'authorize_redirect',
+						'dest_url' => rawurlencode( $dest_url ),
+						'done'     => '1',
 					);
 
+					if ( ! empty( $_GET['from'] ) && 'jetpack_site_only_checkout' === $_GET['from'] ) {
+						// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+						$redirect_args['from'] = $_GET['from'];
```

Comment:
Would we be able to get rid of the PHPCS warning here?

```suggestion
						$redirect_args['from'] = 'jetpack_site_only_checkout';
```

Reply:
Thanks for the review!  Ahh yes.. thank you! 👍